app/build.gradle

```diff
@@ -235,8 +235,14 @@ def gitSha() {
     return res
 }
 
-static def buildTime() {
-    return new Date().format("yyyy-MM-dd'T'HH:mm'Z'", TimeZone.getTimeZone("UTC"))
+def buildTime() {
```

Comment:
The initial idea of this datum was to provide a human readable date of when the project was built.

With the modification this is no longer the case. It now tells about the date of the last modification. This is similar to the commit hash of the last modification. Hence, the function should be renamed to `lastModificationAt`

I understand that a fixed datum is needed to guarantee reproducible builds. I would love to still have the actual build time being included in the - at least - debug builds. In release builds the text can be empty and therefore it can be hidden.

Reply:
I understand. I can take over the pull request once I have to refactor this while still attributing you. Is that okay with you?

Reply:
@eighthave I finally took the time to refactor this. Can you please check and confirm if the implementation will make the build reproducible?!

Reply:
@eighthave I finally took the time to refactor this. :drum: Can you please check and confirm if the implementation will make the build reproducible?!

Reply:
Looks good! Yes, please take this over.